### PR TITLE
GCI-4 - Style Guide Colors Test

### PR DESCRIPTION
**Pull request by Izaron**
Style Guide Colors Test

Ticket: https://issues.openmrs.org/browse/GCI-4
Melange task: https://www.google-melange.com/gci/task/view/google/gci2014/5330144977747968
YouTube Video (Old): http://youtu.be/h2hE8O2HIdQ
YouTube Video (New): http://youtu.be/ZzaEbTduiRY

This code check style guide colors on the pages (Style Guide - http://devtest01.openmrs.org/openmrs/uicommons/styleGuide.page)
Acceptable colors: black, white, grayscale, background colors, colors from Style Guide

As can be seen from the both video, the pages are have so many different shades of red, green, blue and other colors
It is necessary to combine a large number of colors in one shade
Also, there are many shades of gray with small differences, which can be combined in one shade

Last PR: https://github.com/openmrs/openmrs-distro-referenceapplication/pull/37

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/StyleColorTest.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+			String backgroundColor = Color.fromString(item.getCssValue("background-color")).asHex();
+			
+			// check "color" css value
+			if(color != "" && !colors.contains(color)) {
+				System.out.println("Weird color: " + color + ", Item path: " + getElementXPath(driver, item));
+			}
+			
+			// check "background-color" css value
+			if(backgroundColor != "" && !colors.contains(backgroundColor)) {
+				System.out.println("Weird background color: " + backgroundColor + ", Item path: " + getElementXPath(driver, item));
```
Comment (harsha89) [inline ui-tests/src/test/java/org/openmrs/reference/StyleColorTest.java]: Why using soutts? Best way to use logs!

**Comment by Izaron** [comment]
@harsha89 
Java logger use weird timestamp, as on screenshot
Should I use my own log Formatter without timestamp line? So it will be clearer. There is example - http://kodejava.org/how-do-i-create-a-custom-logger-formatter/

timestamp

**Comment by harsha89** [comment]
Can you use 	private static final Log log = LogFactory.getLog(Yourclass.class) and see?

**Comment by Izaron** [comment]
@harsha89 
Thanks, it worked! (See screenshot)
I added log4j.properties, without this file was a warning "log4j:WARN No appenders could be found" and apache Log not working
timestamp

**Comment by djazayeri** [comment]
We do not need any scripted browser tests for the style guide. These slow down the build considerably, and should be used only used to cover real application functionality that we are concerned about regressions in between releases.

**Pull request closed without merging**